charts/osm/Chart.yaml

```
@@ -14,11 +14,11 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.8.4
+version: 0.9.0
```

Same for all instances:
```suggestion
version: 0.9.0-rc.1
```

---

Done.